src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -233,12 +233,12 @@ public bool IsBetween(TimeOnly start, TimeOnly end)
         public static bool operator <=(TimeOnly left, TimeOnly right) => left._ticks <= right._ticks;
 
         /// <summary>
-        /// Subtracts a specified time from another specified time and returns a time interval.
+        ///  Gives the elapsed time between two points on a circular clock, which will always be a positive value.
         /// </summary>
-        /// <param name="t1">The time value to subtract from (the minuend).</param>
-        /// <param name="t2">The time value to subtract (the subtrahend).</param>
-        /// <returns>The absolute time interval between t1 and t2. </returns>
-        public static TimeSpan operator -(TimeOnly t1, TimeOnly t2) => new TimeSpan(Math.Abs(t1._ticks - t2._ticks));
+        /// <param name="t1">The first TimeOnly instance.</param>
+        /// <param name="t2">The second TimeOnly instance..</param>
+        /// <returns>The elapsed time between t1 and t2.</returns>
+        public static TimeSpan operator -(TimeOnly t1, TimeOnly t2) => new TimeSpan((t2._ticks - t1._ticks + TimeSpan.TicksPerDay) % TimeSpan.TicksPerDay);
```

Comment:
I think t1/t2 are swapped here.  I had them wrong in my last edit.

Reply:
I don't think so. it is `-` negative sign. `-` is not from-to.

Reply:
@mattjohnsonpint please let me know what you think about how I am thinking in this operation.

Reply:
The test you have shows how it's inverted.  Yes, 4-1=3, but currently it's doing 1-4=3 which is wrong obviously.

Reply:
talked offline with @mattjohnsonpint and we were saying same thing. I have submitted the fix.